src/views/Vote/Input.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import styled from 'styled-components'
```

Comment:
styled-components should not be used in this code

Reply:
Done